### ISSUE TITLE
Strong-name the assemblies

### ISSUE THREAD

**Issue by JamieMagee** [mthalman/DockerfileModel#101]
Is it possible to strong name[^1] the assemblies? Would you accept a pull request, similar to [this one][1], which does this?

[^1]: https://learn.microsoft.com/en-us/dotnet/standard/assembly/strong-named

[1]: https://github.com/dotnet/Docker.DotNet/pull/567

**Comment by mthalman** [comment]
Apparently I never posted the response that I typed up last month. Let me try again.

What is the use case for having strong-named assemblies? Unlike .NET Framework, strong-named assemblies for .NET (Core) is basically meaningless. See this thread: https://github.com/dotnet/docs/issues/22412

**Comment by JamieMagee** [comment]
I thought it still was recommended for class libraries? I am basing it off [this video][1].

Regardless, this ask comes from an internal team that has a requirement for strong-named assemblies.

[1]: https://www.youtube.com/watch?v=hLFyycJVo0I&t=1471s

**Comment by mthalman** [comment]
No plans to support this.
